### DISCUSSION TITLE
Cancellation form

### DISCUSSION THREAD

**Discussion by MagnusOxlund** [hizzle-co/noptin#941] [Ideas]
I'm working with a client who's interested in offering a cancellation form to reassure potential subscribers that canceling is easy. It's been suggested to place a link to the cancellation form underneath the subscription form as part of the existing note.

Essentially, a cancellation form would work as a backup in case a subscriber has lost all email messages with a cancellation link.

Technically, I imagine a cancellation form would work much like a subscription form:
1. The subscriber would submit their email address.
2. Next, the subscriber would need to confirm the cancellation via email (if the form is configured to require confirmation).

**Comment by picocodes** [comment]
Hi @MagnusOxlund,

You can already create unsubscription forms:- https://noptin.com/guide/subscription-forms/unsubscribe-forms/

If you need to send an unsubscription confirmation email, first [use one of our supported form integrations](https://noptin.com/integrations/) to create a form and then set up a Noptin automated email that is sent to anyone who fills out the form. You can then include the normal Noptin unsubscribe link in this email.

**Reply by MagnusOxlund** [reply]
Thank you for pointing that out. I don't know how I missed the option.
